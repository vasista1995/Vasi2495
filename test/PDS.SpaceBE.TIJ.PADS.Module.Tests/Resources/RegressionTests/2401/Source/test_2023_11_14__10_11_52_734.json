{
  "IdSource": "SPACEACT2:TIJ:BE:104OOIL.I01:Plastic_Trim:Assy_EOL:DambarDisplacement_Distance:1084:3096860:0",
  "Data_flat_Metadata": {
    "PackageFamily": "TO262",
    "Package": "TO262-3",
    "Segment": "Tijuana",
    "Module": "Assy_EOL",
    "FourDReport": "*",
    "ParameterClass": "NON_PPAP",
    "ProcessEquipment": "TRPS0441",
    "Classification": "Setup/Conversi√≥n",
    "Operator": "CSALINAS1",
    "PartNumber": "MA005678599",
    "Track": "MP2010-1-A2-1",
    "Area": "Plastic_Trim",
    "Lot": "104OOIL.I01",
    "ParameterName": "DambarDisplacement_Distance",
    "ParameterUnit": "mils",
    "SiteKey": "TIJ",
    "ChannelId": "1084",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-10T02:59:18Z",
    "CreatedTimestamp": "2023-10-09T19:59:18Z",
    "UpdatedTimestamp": "2023-10-09T19:59:18Z",
    "UpdatedTimestampUtc": "2023-10-10T02:59:18Z",
    "ExportedTimestampUtc": "2023-10-09T19:59:18Z",
    "ExportedTimestamp": "2023-10-09T19:59:18Z",
    "SampleTimestampUtc": "2023-10-10T02:59:18Z",
    "SampleTimestamp": "2023-10-09T19:59:18Z",
    "SourceDataLevel": "C",
    "Equipment": "CMST0006",
    "ChannelName": "DambarDisplacement_Distance/TO262-3",
    "ChannelDescr": "singulation plastics",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/18/2023 09:54:58",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 96472,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2401",
    "IsCalcParam": 0,
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecHigh": 5.0,
        "SpecTarget": 0.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "1,23",
        "Median": "1,2",
        "Sigma": ",238746727726266",
        "Min": "1",
        "Max": "1,6",
        "Q1": "1,05",
        "Q3": "1,3"
      },
      "ControlLimits": {
        "ctrl_mean_low": 0.00430644531069646,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 1.14493693693694,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 2.28556742856318,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "Y",
        "ctrl_range_low": 0.220375690949331,
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.663172362420041,
        "ctrl_range_high": 2.24675905234989,
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 1.13714285714286,
        "ExtEwmaMV": 1.18734863441963,
        "ExtMsMV": 0.0494974746830584,
        "EwmaS": 0.276200008255837,
        "EwmaR": 0.673045141730035,
        "ExtEwmaMVLCL": 0.573328445949387,
        "ExtEwmaMVCenter": 1.14493693693694,
        "ExtEwmaMVUCL": 1.71654542792449,
        "ExtMaMVLCL": 0.712841532796046,
        "ExtMaMVCenter": 1.14493693693694,
        "ExtMaMVUCL": 1.57703234107783,
        "EwmaSLCL": 0.0747247792675185,
        "EwmaSCenter": 0.303626869438673,
        "EwmaSUCL": 0.532528959609827,
        "EwmaRLCL": 0.214522362175703,
        "EwmaRCenter": 0.734234234234235,
        "EwmaRUCL": 1.25394610629277,
        "ExtMSMVLCL": 0.000568000640238796,
        "ExtMSMVCenter": 0.267852048713454,
        "ExtMSMVUCL": 1.07597205270912,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": -1.04067280295617,
        "RawUal": 3.33054667683005,
        "SigmaLal": -0.0572090049142497,
        "SigmaUal": 1.43709298083464,
        "RangeLal": -0.222420980521378,
        "RangeUal": 3.83034574227974
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 3096860,
        "Seqnr": 2,
        "Value": 1.6,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096860,
        "Seqnr": 1,
        "Value": 1.2,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096860,
        "Seqnr": 3,
        "Value": 1.3,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096860,
        "Seqnr": 5,
        "Value": 1.05,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096860,
        "Seqnr": 4,
        "Value": 1.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:TIJ:BE:Plastic_Trim:Assy_EOL:DambarDisplacement_Distance:1084",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:104OOIL.I01",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "TIJ:BE:104OOIL.I01:Plastic_Trim:Assy_EOL:DambarDisplacement_Distance:1084:3096860:0"
  }
}