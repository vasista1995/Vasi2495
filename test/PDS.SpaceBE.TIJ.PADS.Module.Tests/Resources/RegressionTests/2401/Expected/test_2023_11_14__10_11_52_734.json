{
  "Data_flat_Metadata": {
    "Classification": "Setup/Conversi√≥n",
    "PartNumber": "MA005678599",
    "Track": "MP2010-1-A2-1",
    "Area": "Plastic_Trim",
    "MeasLot": "104OOIL.I01",
    "SiteKey": "TIJ",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-10T02:59:18Z",
    "BeginTimestamp": "2023-10-09T19:59:18Z",
    "EndTimestampUtc": "2023-10-10T02:59:18Z",
    "EndTimestamp": "2023-10-09T19:59:18Z"
  },
  "Data_flat_StrangeMetadata": {
    "Module": "Assy_EOL",
    "PackageFamily": "TO262",
    "Package": "TO262-3",
    "Operator": "CSALINAS1"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "process_equipment": "TRPS0441",
          "Seqnr": 2,
          "SampleId": 3096860,
          "Value": 1.6,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:59:18Z",
          "SampleTimestampUtc": "2023-10-10T02:59:18Z",
          "CreatedTimestamp": "2023-10-09T19:59:18Z",
          "CreatedTimestampUtc": "2023-10-10T02:59:18Z",
          "UpdatedTimestamp": "2023-10-09T19:59:18Z",
          "UpdatedTimestampUtc": "2023-10-10T02:59:18Z",
          "NumViolations": 0,
          "SampleMean": 1.23,
          "SampleMin": 1.0,
          "SampleMax": 1.6,
          "SampleMedian": 1.2,
          "SampleStdev": 0.238746727726266,
          "SampleQ1": 1.05,
          "SampleQ3": 1.3,
          "SampleSize": 5
        },
        {
          "process_equipment": "TRPS0441",
          "Seqnr": 1,
          "SampleId": 3096860,
          "Value": 1.2,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:59:18Z",
          "SampleTimestampUtc": "2023-10-10T02:59:18Z",
          "CreatedTimestamp": "2023-10-09T19:59:18Z",
          "CreatedTimestampUtc": "2023-10-10T02:59:18Z",
          "UpdatedTimestamp": "2023-10-09T19:59:18Z",
          "UpdatedTimestampUtc": "2023-10-10T02:59:18Z",
          "NumViolations": 0,
          "SampleMean": 1.23,
          "SampleMin": 1.0,
          "SampleMax": 1.6,
          "SampleMedian": 1.2,
          "SampleStdev": 0.238746727726266,
          "SampleQ1": 1.05,
          "SampleQ3": 1.3,
          "SampleSize": 5
        },
        {
          "process_equipment": "TRPS0441",
          "Seqnr": 3,
          "SampleId": 3096860,
          "Value": 1.3,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:59:18Z",
          "SampleTimestampUtc": "2023-10-10T02:59:18Z",
          "CreatedTimestamp": "2023-10-09T19:59:18Z",
          "CreatedTimestampUtc": "2023-10-10T02:59:18Z",
          "UpdatedTimestamp": "2023-10-09T19:59:18Z",
          "UpdatedTimestampUtc": "2023-10-10T02:59:18Z",
          "NumViolations": 0,
          "SampleMean": 1.23,
          "SampleMin": 1.0,
          "SampleMax": 1.6,
          "SampleMedian": 1.2,
          "SampleStdev": 0.238746727726266,
          "SampleQ1": 1.05,
          "SampleQ3": 1.3,
          "SampleSize": 5
        },
        {
          "process_equipment": "TRPS0441",
          "Seqnr": 5,
          "SampleId": 3096860,
          "Value": 1.05,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:59:18Z",
          "SampleTimestampUtc": "2023-10-10T02:59:18Z",
          "CreatedTimestamp": "2023-10-09T19:59:18Z",
          "CreatedTimestampUtc": "2023-10-10T02:59:18Z",
          "UpdatedTimestamp": "2023-10-09T19:59:18Z",
          "UpdatedTimestampUtc": "2023-10-10T02:59:18Z",
          "NumViolations": 0,
          "SampleMean": 1.23,
          "SampleMin": 1.0,
          "SampleMax": 1.6,
          "SampleMedian": 1.2,
          "SampleStdev": 0.238746727726266,
          "SampleQ1": 1.05,
          "SampleQ3": 1.3,
          "SampleSize": 5
        },
        {
          "process_equipment": "TRPS0441",
          "Seqnr": 4,
          "SampleId": 3096860,
          "Value": 1.0,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:59:18Z",
          "SampleTimestampUtc": "2023-10-10T02:59:18Z",
          "CreatedTimestamp": "2023-10-09T19:59:18Z",
          "CreatedTimestampUtc": "2023-10-10T02:59:18Z",
          "UpdatedTimestamp": "2023-10-09T19:59:18Z",
          "UpdatedTimestampUtc": "2023-10-10T02:59:18Z",
          "NumViolations": 0,
          "SampleMean": 1.23,
          "SampleMin": 1.0,
          "SampleMax": 1.6,
          "SampleMedian": 1.2,
          "SampleStdev": 0.238746727726266,
          "SampleQ1": 1.05,
          "SampleQ3": 1.3,
          "SampleSize": 5
        }
      ],
      "process_equipment": "TRPS0441",
      "FourDReport": "*",
      "ParameterClass": "NON_PPAP",
      "Segment": "Tijuana",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_high": 5.0,
          "target": 0.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 0.00430644531069646,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 1.14493693693694,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 2.28556742856318,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "Y",
          "CtrlSigmaCenterEnabled": "Y",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "Y",
          "ctrl_range_low": 0.220375690949331,
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 0.663172362420041,
          "ctrl_range_high": 2.24675905234989,
          "CtrlRangeHighEnabled": "N"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 1.23,
        "median": 1.2,
        "stdev": 0.21354156504062627,
        "range": 0.6000000000000001,
        "min": 1.0,
        "max": 1.6,
        "q02": 1.004,
        "q05": 1.01,
        "q25": 1.05,
        "q75": 1.3,
        "q95": 1.54,
        "q98": 1.576,
        "NumViolations": 0,
        "Samples": "3096860"
      },
      "ParameterName": "DambarDisplacement_Distance",
      "ParameterUnit": "mils",
      "ChannelId": "1084",
      "SourceDataLevel": "C",
      "channel_name": "DambarDisplacement_Distance TO262-3",
      "channel_description": "singulation plastics",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-10T02:59:18Z",
      "created_timestamp": "2023-10-09T19:59:18Z",
      "updated_timestamp_utc": "2023-10-10T02:59:18Z",
      "updated_timestamp": "2023-10-09T19:59:18Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-10-10T02:59:18Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:TIJ:BE:Plastic_Trim:Assy_EOL",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:104OOIL.I01",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "TIJ:BE:104OOIL.I01:Plastic_Trim:Assy_EOL:DambarDisplacement_Distance:1084:3096860:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-09T19:59:18Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:104OOIL.I01:SPACEAGGED2:TIJ:BE:Plastic_Trim:Assy_EOL:ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "TIJ"
  }
}